hypervisor/dm/vpci/vpci_bridge.c

```diff
@@ -50,6 +50,38 @@
 #include <pci.h>
 #include "vpci_priv.h"
 
+/**
+ * @addtogroup vp-dm_vperipheral
+ *
+ * @{
+ */
+
+/**
+ * @file
+ * @brief Implementation of virtual pci bridge.
+ *
+ * This file defines operations to support virtual pci bridge. It implements struct pci_vdev_ops and related functions.
+ */
+
+/**
+ * @brief Initializes the vPCI bridge.
+ *
+ * A PCI bridge is also a PCI device. This function initializes the specified virtual PCI device as a PCI bridge. A vPCI
+ * bridge is based on a physical PCI bridge and is used for Service VM. It's usually used in the initialization phase of
+ * Service VM.
+ *
+ * It initializes all the PCI configuration space registers from the physical PCI bridge and emulates the type info.
```

Comment:
As the `type info` is a little bit vague, I would suggest say more details here to define them.
Some suggestions from my side:
`
It initializes most of the virtual PCI configuration space registers based on the physical PCI bridge registers, except those that specify the type information, which is emulated.
Such type related registers include Vendor ID, Device ID, Revision ID, Header Type, class and sub-class code.
`